<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>黑羽数据管理</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/数据分析.png</normaloff>images/数据分析.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="columnCount">
         <number>1</number>
        </property>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QMdiArea" name="mdiArea"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>测试</string>
    </property>
    <addaction name="actionloadtest"/>
   </widget>
   <widget class="QMenu" name="menuguanyu">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionversion"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>运维</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>数据分析</string>
    </property>
    <addaction name="action_data_analysis"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>配置</string>
    </property>
    <addaction name="actionconnectset"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>导入导出</string>
    </property>
    <addaction name="actionImportAndExport"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
   <addaction name="menuguanyu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionconnectset"/>
   <addaction name="actionexit"/>
   <addaction name="actionloadtest"/>
   <addaction name="action_data_analysis"/>
  </widget>
  <action name="actionloadtest">
   <property name="icon">
    <iconset>
     <normaloff>images/load.png</normaloff>images/load.png</iconset>
   </property>
   <property name="text">
    <string>压力测试</string>
   </property>
  </action>
  <action name="actionversion">
   <property name="text">
    <string>版本号</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="icon">
    <iconset>
     <normaloff>images/exit (2).png</normaloff>images/exit (2).png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionconnectset">
   <property name="icon">
    <iconset>
     <normaloff>images/set (2).png</normaloff>images/set (2).png</iconset>
   </property>
   <property name="text">
    <string>配置连接参数</string>
   </property>
  </action>
  <action name="actionImportAndExport">
   <property name="text">
    <string>导入导出</string>
   </property>
  </action>
  <action name="action_data_analysis">
   <property name="icon">
    <iconset>
     <normaloff>images/数据.png</normaloff>images/数据.png</iconset>
   </property>
   <property name="text">
    <string>数据分析</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
